src/app.js

```diff
@@ -1,5 +1,190 @@
 const express = require('express');
 
+const {
+  sayHello,
+  uppercase,
+  lowercase,
+  countCharacters,
+  firstCharacter,
+  firstCharacters,
+} = require('./lib/strings');
+const { add, subtract, multiply, divide, remainder } = require('./lib/numbers');
+const { negate, truthiness, isOdd, startsWith } = require('./lib/booleans');
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray2,
+  elementsStartingWithAVowel,
+  removeNthElement2,
+} = require('./lib/arrays');
+
 const app = express();
+app.use(express.json());
+
+app.get('/strings/hello/:string', (req, res) => {
+  res.json({ result: sayHello(req.params.string) });
+});
+
+app.get('/strings/upper/:string', (req, res) => {
+  res.json({ result: uppercase(req.params.string) });
+});
+
+app.get('/strings/lower/:string', (req, res) => {
+  res.json({ result: lowercase(req.params.string) });
+});
+
+app.get('/strings/count-characters/:string', (req, res) => {
+  res.json({ result: countCharacters(req.params.string) });
+});
+
+app.get('/strings/first-character/:string', (req, res) => {
+  res.status(200).json({ result: firstCharacter(req.params.string) });
+});
+
+app.get('/strings/first-characters/:string', (req, res) => {
+  const qlength = req.query.length;
```

Comment:
I am not sure why you are assigning the query length to a variable, but you dont do the same to the req.params.string. :)
Both or none would be better for consistency.